src/server/api/lib/import-script.js

```diff
@@ -336,7 +337,17 @@ const makeCannedResponsesList = cannedResponsesParagraphs => {
       !cannedResponsesParagraphs[0].isParagraphBold
     ) {
       const textParagraph = cannedResponsesParagraphs.shift();
-      cannedResponse.text.push(textParagraph.text);
+      if (textParagraph.isParagraphItalic) {
+        // Italic = tag.
+        const tagId = textParagraph.text.match(/^\d*\b/);
+        if (!!tagId[0]) {
```

Comment:
this will fail when there is no match with a type error null can't index [0] == tweak condition to `if (tagId)`

Reply:
Are you sure? When I was testing, `match()` always returns an array.
E.g.: `'foo'.match(/^\d*\b/)`
Returns: `[ '', index: 0, input: 'foo', groups: undefined ]`

Reply:
@schuyler1d - I couldn't reproduce the error you mentioned, but I added the extra condition just to be safe. The `!!tagId[0]` needs to be there as well. Without it, when you try to import a script with a non-matching tag, you get an error:
`insert into "tag_canned_response" ("canned_response_id", "tag_id") values ($1, $2), ($3, $4), ($5, $6) - invalid input syntax for type integer: ""`

Reply:
```
$ nvm use
Found '/home/sky/sb/moveon/spoke/.nvmrc' with version <10>
Now using node v10.19.0 (npm v6.13.4)
$ node
> 'asdf'.match(/xxx/)
null
> 'asdf'.match(/asd/)
[ 'asd', index: 0, input: 'asdf', groups: undefined ]
> 'asdf'.match(/as(d)/)
[ 'asd', 'd', index: 0, input: 'asdf', groups: undefined ]
```

in browser: https://jsfiddle.net/3uwj8La1/